tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "target": "ES6",
+    "allowJs": true,
+    "checkJs": true,
+    "noEmit": true,
+    "outDir": "./dist"
```

Comment:
Would the `tsc` fit somewhere in the build process?

Reply:
Yeah. You could link it into the build process. My thought was to make it part of the tests along with `eslint` and make the tests fail if any errors are detected. Though we'd have to fix all the errors before enabling that. We might be able to turn it on just for a file or directory at the beginning until the others are fixed